Kernel/FileSystem/DevPtsFS.cpp

```diff
@@ -59,10 +59,12 @@ ErrorOr<NonnullRefPtr<Inode>> DevPtsFS::get_inode(InodeIdentifier inode_id) cons
         return *m_root_inode;
 
     unsigned pty_index = inode_index_to_pty_index(inode_id.index());
-    auto* device = DeviceManagement::the().get_device(201, pty_index);
+    auto device = DeviceManagement::the().get_device(false, 201, pty_index);
     VERIFY(device);
+    // FIXME: Find a better way to cast here
+    RefPtr<SlavePTY> pty_device = static_cast<SlavePTY*>(device.ptr());
```

Comment:
Do you mean something like this?
```suggestion
    auto pty_device = static_ptr_cast<SlavePTY>(device);
```

Reply:
Yes. Thank you :)